no sql function layer_{layername} after following the tutorial

# Description

I followed the tutorial, but when I tried to serve my tiles, postile shows these logs

```
[2019-07-25 10:39:41 +0200] [6293] [ERROR] Exception occurred while handling uri: "http://localhost:8090/6/31/24.pbf"
Traceback (most recent call last):
  File "/home/augustin/workspace/postile/venv/lib/python3.6/site-packages/sanic/app.py", line 603, in handle_request
    response = await response
  File "/usr/lib/python3.6/asyncio/coroutines.py", line 110, in __next__
    return self.gen.send(None)
  File "/home/augustin/workspace/postile/postile/postile/postile.py", line 138, in get_tile_tm2
    rows = await conn.fetch(sql)
  File "/usr/lib/python3.6/asyncio/coroutines.py", line 129, in throw
    return self.gen.throw(type, value, traceback)
  File "/home/augustin/workspace/postile/venv/lib/python3.6/site-packages/asyncpg/connection.py", line 421, in fetch
    return await self._execute(query, args, 0, timeout)
  File "/usr/lib/python3.6/asyncio/coroutines.py", line 129, in throw
    return self.gen.throw(type, value, traceback)
  File "/home/augustin/workspace/postile/venv/lib/python3.6/site-packages/asyncpg/connection.py", line 1414, in _execute
    query, args, limit, timeout, return_status=return_status)
  File "/usr/lib/python3.6/asyncio/coroutines.py", line 129, in throw
    return self.gen.throw(type, value, traceback)
  File "/home/augustin/workspace/postile/venv/lib/python3.6/site-packages/asyncpg/connection.py", line 1422, in __execute
    return await self._do_execute(query, executor, timeout)
  File "/usr/lib/python3.6/asyncio/coroutines.py", line 129, in throw
    return self.gen.throw(type, value, traceback)
  File "/home/augustin/workspace/postile/venv/lib/python3.6/site-packages/asyncpg/connection.py", line 1434, in _do_execute
    stmt = await self._get_statement(query, None)
  File "/usr/lib/python3.6/asyncio/coroutines.py", line 129, in throw
    return self.gen.throw(type, value, traceback)
  File "/home/augustin/workspace/postile/venv/lib/python3.6/site-packages/asyncpg/connection.py", line 329, in _get_statement
    statement = await self._protocol.prepare(stmt_name, query, timeout)
  File "asyncpg/protocol/protocol.pyx", line 163, in prepare
asyncpg.exceptions.UndefinedFunctionError: function layer_waterway(box2d, integer) does not exist
HINT:  No function matches the given name and argument types. You might need to add explicit type casts.

```

I can't find the steps when these functions are created in the tutorial.

Thanks!

--- a/README.md
+++ b/README.md
@@ -43,10 +43,9 @@ NB: **postgis >= 2.4.0** is required (with st_asmvt* functions)
     create extension fuzzystrmatch;
     create extension osml10n;
 
-We have to find the Docker Gateway IP to be able to connect from the container to the host:
+Storing the gateway IP of the Docker bridge interface to be able to connect from the container to our local PostgreSQL:
 
-    $ docker network inspect bridge | jq .[0].IPAM.Config[0].Gateway
-    "172.17.0.1"
+    $ export PG_GATEWAY=$(docker network inspect bridge | jq -r .[0].IPAM.Config[0].Gateway)
 
 Change the PostgreSQL configuration to allow interacting with container's tools:
 
@@ -68,21 +67,21 @@ and the [OpenMapTiles README](https://github.com/openmaptiles/openmaptiles)
 2. Download and load OSM dataset (example used is the Rhône-Alpes area in France)
    
         wget -P data/ http://download.geofabrik.de/europe/france/rhone-alpes-latest.osm.pbf
-        docker run --rm -v $(pwd)/data:/import -v $(pwd)/build:/mapping -e POSTGRES_DB="osm" -e POSTGRES_PORT="5432" -e POSTGRES_HOST="172.17.0.1" -e POSTGRES_PASSWORD="osm" -e POSTGRES_USER="osm" openmaptiles/import-osm:0.5
+        docker run --rm -v $(pwd)/data:/import -v $(pwd)/build:/mapping -e POSTGRES_DB="osm" -e POSTGRES_PORT="5432" -e POSTGRES_HOST=$PG_GATEWAY -e POSTGRES_PASSWORD="osm" -e POSTGRES_USER="osm" openmaptiles/import-osm:0.5
         docker run --rm -v $(pwd)/data:/import openmaptiles/generate-osmborder
-        docker run --rm -v $(pwd)/data:/import -e POSTGRES_DB="osm" -e POSTGRES_PORT="5432" -e POSTGRES_HOST="172.17.0.1" -e POSTGRES_PASSWORD="osm" -e POSTGRES_USER="osm" openmaptiles/import-osmborder:0.4
+        docker run --rm -v $(pwd)/data:/import -e POSTGRES_DB="osm" -e POSTGRES_PORT="5432" -e POSTGRES_HOST=$PG_GATEWAY -e POSTGRES_PASSWORD="osm" -e POSTGRES_USER="osm" openmaptiles/import-osmborder:0.4
 
 3. Download additional non-OSM data
 
-        docker run --rm -e POSTGRES_DB="osm" -e POSTGRES_PORT="5432" -e POSTGRES_HOST="172.17.0.1" -e POSTGRES_PASSWORD="osm" -e POSTGRES_USER="osm" openmaptiles/import-natural-earth:1.4
-        docker run --rm -e POSTGRES_DB="osm" -e POSTGRES_PORT="5432" -e POSTGRES_HOST="172.17.0.1" -e POSTGRES_PASSWORD="osm" -e POSTGRES_USER="osm" openmaptiles/import-lakelines:1.0
-        docker run --rm -e POSTGRES_DB="osm" -e POSTGRES_PORT="5432" -e POSTGRES_HOST="172.17.0.1" -e POSTGRES_PASSWORD="osm" -e POSTGRES_USER="osm" openmaptiles/import-water:1.1
+        docker run --rm -e POSTGRES_DB="osm" -e POSTGRES_PORT="5432" -e POSTGRES_HOST=$PG_GATEWAY -e POSTGRES_PASSWORD="osm" -e POSTGRES_USER="osm" openmaptiles/import-water:1.1
+        docker run --rm -e POSTGRES_DB="osm" -e POSTGRES_PORT="5432" -e POSTGRES_HOST=$PG_GATEWAY -e POSTGRES_PASSWORD="osm" -e POSTGRES_USER="osm" openmaptiles/import-natural-earth:1.4
+        docker run --rm -e POSTGRES_DB="osm" -e POSTGRES_PORT="5432" -e POSTGRES_HOST=$PG_GATEWAY -e POSTGRES_PASSWORD="osm" -e POSTGRES_USER="osm" openmaptiles/import-lakelines:1.0
 
 4. Grab some wikipedia data used for translations (WARNING: this part can take a while since the file to download is ~30GB)
-   If you don't care about translations, you can skip this part.
+   If you don't care about translations, you can skip the first command, the second one will create the empty relations for next steps.
 
         docker run --rm -v $(pwd)/wikidata:/import --entrypoint /usr/src/app/download-gz.sh openmaptiles/import-wikidata:0.1
-        docker run --rm -v $(pwd)/wikidata:/import -e POSTGRES_DB="osm" -e POSTGRES_PORT="5432" -e POSTGRES_HOST="172.17.0.1" -e POSTGRES_PASSWORD="osm" -e POSTGRES_USER="osm" openmaptiles/import-wikidata:0.1
+        docker run --rm -v $(pwd)/wikidata:/import -e POSTGRES_DB="osm" -e POSTGRES_PORT="5432" -e POSTGRES_HOST=$PG_GATEWAY -e POSTGRES_PASSWORD="osm" -e POSTGRES_USER="osm" openmaptiles/import-wikidata:0.1
 
 5. Load sql wrappers used for rendering
    

--- a/README.md
+++ b/README.md
@@ -86,7 +86,7 @@ and the [OpenMapTiles README](https://github.com/openmaptiles/openmaptiles)
 
 5. Load sql wrappers used for rendering
    
-        docker run --rm -v $(pwd)/build:/sql -e POSTGRES_DB="osm" -e POSTGRES_PORT="5432" -e POSTGRES_HOST="172.17.0.1" -e POSTGRES_PASSWORD="osm" -e POSTGRES_USER="osm" openmaptiles/import-sql:0.7
+        docker run --rm -v $(pwd)/build:/sql -e POSTGRES_DB="osm" -e POSTGRES_PORT="5432" -e POSTGRES_HOST="172.17.0.1" -e POSTGRES_PASSWORD="osm" -e POSTGRES_USER="osm" openmaptiles/import-sql:0.8
 
 ## Choose a Mapbox GL style
 

--- a/README.md
+++ b/README.md
@@ -74,9 +74,9 @@ and the [OpenMapTiles README](https://github.com/openmaptiles/openmaptiles)
 
 3. Download additional non-OSM data
 
-        docker run --rm -e POSTGRES_DB="osm" -e POSTGRES_PORT="5432" -e POSTGRES_HOST="172.17.0.1" -e POSTGRES_PASSWORD="osm" -e POSTGRES_USER="osm" openmaptiles/import-water:0.6
         docker run --rm -e POSTGRES_DB="osm" -e POSTGRES_PORT="5432" -e POSTGRES_HOST="172.17.0.1" -e POSTGRES_PASSWORD="osm" -e POSTGRES_USER="osm" openmaptiles/import-natural-earth:1.4
         docker run --rm -e POSTGRES_DB="osm" -e POSTGRES_PORT="5432" -e POSTGRES_HOST="172.17.0.1" -e POSTGRES_PASSWORD="osm" -e POSTGRES_USER="osm" openmaptiles/import-lakelines:1.0
+        docker run --rm -e POSTGRES_DB="osm" -e POSTGRES_PORT="5432" -e POSTGRES_HOST="172.17.0.1" -e POSTGRES_PASSWORD="osm" -e POSTGRES_USER="osm" openmaptiles/import-water:1.1
 
 4. Grab some wikipedia data used for translations (WARNING: this part can take a while since the file to download is ~30GB)
    If you don't care about translations, you can skip this part.

--- a/README.md
+++ b/README.md
@@ -58,13 +58,12 @@ Change the PostgreSQL configuration to allow interacting with container's tools:
 Extracted from [docker-compose.yml](https://github.com/openmaptiles/openmaptiles/blob/master/docker-compose.yml) 
 and the [OpenMapTiles README](https://github.com/openmaptiles/openmaptiles) 
 
-1. Download tools to generate tm2source for all layers
+1. Build the imposm mapping, the tm2source project and collect all SQL scripts
 
-        pip install openmaptiles-tools  # inside a python 2 venv
-        git clone git@github.com:openmaptiles/openmaptiles.git
-        cd openmaptiles
-        # make the imposm3 mapping, tm2source file and aggregate SQL layers 
-        make
+        mkdir -p build/openmaptiles.tm2source
+        docker run -v $(pwd):/tileset -v $(pwd)/build:/sql --rm openmaptiles/openmaptiles-tools generate-tm2source openmaptiles.yaml --port=5432 --database="osm" --user="osm" --password="osm" > build/openmaptiles.tm2source/data.yml
+        docker run -v $(pwd):/tileset -v $(pwd)/build:/sql --rm openmaptiles/openmaptiles-tools generate-imposm3 openmaptiles.yaml > build/mapping.yaml
+        docker run -v $(pwd):/tileset -v $(pwd)/build:/sql --rm openmaptiles/openmaptiles-tools generate-sql openmaptiles.yaml > build/tileset.sql
 
 2. Download and load OSM dataset (example used is the Rhône-Alpes area in France)
    

